### DISCUSSION TITLE
Importing .osc file for EBSD plotting

### DISCUSSION THREAD

**Discussion by ute23** [mtex-toolbox/mtex#1936] [Ask Anything]
I am importing a large .osc file(164355 orientations) into MTEX but only get the scalebar when plotting ebsd.

**Comment by lfgmorales** [comment]
Hi

I suppose you will need to be a bit more verbose in explaining what the problem is and what do you want to do

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 31 Aug 2023, at 00:05, ute23 ***@***.***> wrote:

﻿

I am importing a large .osc file(164355 orientations) into MTEX but only get the scalebar when plotting ebsd.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1936>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2QH4PIG6QR3K64LEDXX62LVANCNFSM6AAAAAA4FE6VWY>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1936", "url": "https://github.com/mtex-toolbox/mtex/discussions/1936", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by ute23** [comment]
AM 3-19_post fatigue_11Dec2211_1200X.osc
<https://drive.google.com/file/d/1weJ7eF3CWDrjoA1Aswm6qMJqtAkYBJ9_/view?usp=drive_web>
Hello Luiz,

When I import the attached .osc google link and then plot(ebsd), I get the
attached figure. I imported it with the Import Wizard and selected "Not
Indexed". There are 164355 orientations but there are no orientations in
the EBSD matrix.  So, when I plot(ebsd, ebsd.orientations) there is an
error.   It is a Ti6Al4V EBSD image generated from EDAX.

[image: image.png]
I also have an OIM file of the same EBSD image from EDAX, but not sure how
to import that into MTEX.

Best,
Roger Beal

**Reply by kilir** [reply]
Mtex does not read oim files and .osc only if the loader in mtex does understand the osc version you have.
Whjat kind of software version did you use to creat e the .osc? 

Usually it's a good idea to use a human readable file format such as an .ang for example.

Cheers,
Rüdiger

**Comment by ute23** [comment]
I used the OIM software that came with the EDAX at the University of Utah.
How can I tell if MTEX accepts the .OSC file? I don't have access to the
equipment anymore.  Is there a way to convert .OSC to ANG? Can you read the
file I sent?

Best,
Roger
